workers/scanner.go

```diff
@@ -327,7 +328,12 @@ func (s *Scanner) managedFindAndPinOneUnderpinnedSkylink() (skylink skymodules.S
 	minPinners := s.minPinners
 	s.mu.Unlock()
 
-	sl, err := s.staticDB.FindAndLockUnderpinned(context.TODO(), s.staticServerName, minPinners)
+	ctx := context.TODO()
+
+	// skipSkylinks is a list of skylinks which we want to skip during this
+	// scan. These might be skylinks which errored out or blocked skylinks.
+	var skipSkylinks []string
```

Comment:
I might just be missing it but I don't see how this is being used. 

You initialize it here and immediately call `FindAndLockUnderpinned` with it, but it is empty at that point. 

You then add skylinks to it, but you never call out to another function with it or make a DB call to update skipped links. So how does the server know about these skipped links the next time around?

Reply:
Right, good catch. Distraction strikes again -I thought i was putting it in the longer running upper method.

I'll move it to the scanner object itself and I'll wipe it on each scan start.